app/scripts/lib/oauth-client.js

```diff
@@ -39,8 +39,6 @@ function (xhr, OAuthErrors) {
 
     // params = { assertion, client_id, scope }
     getToken: function (params) {
-      /* jshint camelcase: false */
-
```

Comment:
@TDA, interesting, it looks like I disabled the `camelcase` config in the root `eslint-config-fxa` module (see https://github.com/pdehaan/eslint-config-fxa/blob/master/index.js#L9). We can enable it for the "fxa/client" template and set it to `2` (error), and when I try that locally it looks like we get 19 errors which we may need to inline ignore for those specific lines/blocks:

```sh
$ grunt eslint
Running "eslint:grunt" (eslint) task

grunttasks/clean.js
  7:4  error  Identifier 'tos_pp' is not in camel case  camelcase

grunttasks/copy.js
  25:4  error  Identifier 'tos_pp' is not in camel case       camelcase
  49:4  error  Identifier 'error_pages' is not in camel case  camelcase

grunttasks/l10n-generate-pages.js
  113:10  error  Identifier 'lang_dir' is not in camel case  camelcase

grunttasks/marked.js
  23:4  error  Identifier 'tos_pp' is not in camel case  camelcase

grunttasks/po2json.js
  17:6  error  Identifier 'output_filename' is not in camel case   camelcase
  45:6  error  Identifier 'output_transform' is not in camel case  camelcase

grunttasks/replace.js
  7:4  error  Identifier 'tos_pp' is not in camel case  camelcase

grunttasks/usemin.js
   7:4  error  Identifier 'client_rendered' is not in camel case  camelcase
  15:4  error  Identifier 'server_rendered' is not in camel case  camelcase

grunttasks/yeoman.js
  14:4  error  Identifier 'page_template_src' is not in camel case   camelcase
  15:4  error  Identifier 'page_template_dist' is not in camel case  camelcase
  16:4  error  Identifier 'strings_src' is not in camel case         camelcase
  17:4  error  Identifier 'strings_dist' is not in camel case        camelcase
  21:4  error  Identifier 'tos_pp_repo_dest' is not in camel case    camelcase
  22:4  error  Identifier 'tos_md_src' is not in camel case          camelcase
  23:4  error  Identifier 'tos_html_dest' is not in camel case       camelcase
  24:4  error  Identifier 'pp_md_src' is not in camel case           camelcase
  25:4  error  Identifier 'pp_html_dest' is not in camel case        camelcase

✖ 19 problems (19 errors, 0 warnings)

Warning: Task "eslint:grunt" failed. Use --force to continue.

Aborted due to warnings.
```